src/main/java/uk/gov/companieshouse/api/accounts/model/rest/DirectorsReport.java

```diff
@@ -0,0 +1,76 @@
+package uk.gov.companieshouse.api.accounts.model.rest;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Map;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class DirectorsReport extends RestObject {
+
+    @JsonProperty("company_policy_on_disabled_employees")
+    private String companyPolicyOnDisabledEmployees;
+
+    @JsonProperty("directors")
+    private Map<String, String> directorsEntity;
```

Comment:
Thinking about it, it may be worth renaming these as just `directors` and `secretaries`? More a matter of personal pref